### PR TITLE
Displaying non-default columns for local .xel files

### LINKED ISSUE #25906: Non-default fields in .xel files are not displayed
Type: <b>Bug</b>

A .xel file produced outside of SQL Server may contain a different set of columns. Since Azure Data Studio's SQL Server Profiler has its own set of display columns, these non-default ones are simply ignored.

Extension version: 0.14.0
Azure Data Studio version: azuredatastudio 1.49.0 (6e6d22ddd74bbb4fbe344ed80b93b53489fe0239, 2024-08-01T05:19:02.202Z)
OS version: Windows_NT x64 10.0.22631
Restricted Mode: No
Preview Features: Enabled
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1270P (16 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|47.67GB (24.80GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

### PR DESCRIPTION
**Pull request by glaubercini**
A .xel file produced outside of SQL Server may contain a different set of columns. Since Azure Data Studio's SQL Server Profiler has its own set of display columns, these non-default ones are simply ignored.

This PR adds support for displaying non-default columns in the details section while preserving the default behavior that fixes #25906 issue.

Before:
before

After:
after

### CODE REVIEW

**Comment by SakshiS-harma** [comment]
Hey @glaubercini, it was great working with you for this bug+fix. 
Do you mind adding a snippet of before (how the data wasn't visible on UI in ADS, but it was presented in SSMS) and after (data presented in ADS as well)? 
Changes look good to me :shipit:  . 
May be just wait for another reviewer to approve before merging your changes

**Review by SakshiS-harma — APPROVED** [review]

**Comment by glaubercini** [comment]
@SakshiS-harma here follow the images:

Before:
before

After:
after

**Review by kburtram — APPROVED** [review]

**Pull request merged**
